index.js

```diff
@@ -13,6 +13,10 @@
  *
  *   Converts straight double and single quotes to smart double or single
  *   quotes.
+ * @property {{ '"': string; "'": string }} [openingQuotes]
```

Comment:
Good to define this object in the types instead of inline.
That way a) other people can use it, b) we can reuse it.

Also idea: call it first and second or something instead of using non-identifier names as keys?

Reply:
i went for `{ double: '”', single: '’' }` - or would you prefer `{ first: '”', second: '’' }`?